openapi-v1.yaml

```diff
@@ -3998,6 +4007,32 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /task/{id}/result:
+    parameters:
+      - name: id
+        in: path
+        description: task id to retrieve stored results
+        type: string
+        required: true
+    get:
+      tags:
+        - tasks
+      description: Retrieve results of an array task 
+      responses:
+        200:
```

Comment:
We should also return 204 for tasks which are not finished but are in the system. Perhaps also explicitly add 404s for IDs that do not exist? I know normally we do not add the error cases but this route is likely to be polled a lot so I believe explicitly having the "expected" types will make the client side try/catch/rerun cycles a little easier with the auto-generated classes.

Reply:
Added the 404 and 202 status codes to the spec